### PR TITLE
`jsObject` deprecation in favour of `jso`

### PR DESCRIPTION
**Pull request by aerialist7**
cc @turansky @sgrishchenko

### CODE REVIEW

File: kotlin-extensions/src/main/kotlin/kotlinext/js/Helpers.kt [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
+    replaceWith = ReplaceWith("jso", "kotlinext.js"),
+)
+@Suppress("NOTHING_TO_INLINE")
+inline fun <T : Any> jsObject(): T =
+    jso()
+
+inline fun <T : Any> jso(builder: T.() -> Unit): T =
+    jso<T>().apply(builder)
+
+@Deprecated(
```
Comment (turansky) [inline kotlin-extensions/src/main/kotlin/kotlinext/js/Helpers.kt:18]: After next release please

File: kotlin-extensions/src/main/kotlin/kotlinext/js/Helpers.kt [hunk 2]
```diff
@@ -1,15 +1,30 @@
 package kotlinext.js
 
 @Suppress("NOTHING_TO_INLINE")
-inline fun <T : Any> jsObject(): T =
+inline fun <T : Any> jso(): T =
     js("({})")
 
+@Deprecated(
```
Comment (turansky) [inline kotlin-extensions/src/main/kotlin/kotlinext/js/Helpers.kt:7]: After next release please

File: kotlin-extensions/src/main/kotlin/kotlinext/js/Helpers.kt [hunk 3]
```diff
@@ -1,15 +1,30 @@
...
+    message = "Legacy type alias",
+    replaceWith = ReplaceWith("jso", "kotlinext.js"),
+)
+@Suppress("NOTHING_TO_INLINE")
```
Comment (turansky) [inline kotlin-extensions/src/main/kotlin/kotlinext/js/Helpers.kt:11]: On file please

**Review by turansky — CHANGES_REQUESTED** [review]

File: kotlin-extensions/src/main/kotlin/kotlinext/js/Helpers.kt (hunk 1 above)
Comment (aerialist7) [reply kotlin-extensions/src/main/kotlin/kotlinext/js/Helpers.kt]: Ok

File: kotlin-extensions/src/main/kotlin/kotlinext/js/Helpers.kt (hunk 2 above)
Comment (aerialist7) [reply kotlin-extensions/src/main/kotlin/kotlinext/js/Helpers.kt]: Sure

File: kotlin-extensions/src/main/kotlin/kotlinext/js/Helpers.kt (hunk 3 above)
Comment (aerialist7) [reply kotlin-extensions/src/main/kotlin/kotlinext/js/Helpers.kt]: Done

File: examples/src/main/kotlin/example/main.kt [hunk 4]
```diff
@@ -1,10 +1,17 @@
...
+import kotlinx.html.DIV
 import kotlinx.html.InputType
-import react.*
+import kotlinx.html.tabIndex
+import kotlinx.html.unsafe
+import react.PropsWithChildren
+import react.RBuilder
+import react.createContext
 import react.dom.*
+import kotlin.collections.set
```
Comment (turansky) [inline examples/src/main/kotlin/example/main.kt:14]: Redundant import?

**Review by turansky — APPROVED** [review]

File: examples/src/main/kotlin/example/main.kt (hunk 4 above)
Comment (aerialist7) [reply examples/src/main/kotlin/example/main.kt:14]: Seems yes

**Comment by Leonya** [comment]
Released in pre.240

**Pull request merged**